### PR TITLE
Exchange inflow signal realtime v2

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Summary

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: priv/repo/migrations/20181128153823_create-signals-cooldown-table.exs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+defmodule :"Elixir.Sanbase.Repo.Migrations.Create-signals-cooldown-table" do
```
[the remark this answers is not in the document]
Comment (IvanIvanoff) [reply priv/repo/migrations/20181128153823_create-signals-cooldown-table.exs]: I did a rollback but forgot to remove the file, so it was introduced again. Now it's fixed.

File: config/notifications_config.exs [hunk 2]
```diff
@@ -31,7 +31,7 @@ config :sanbase, Sanbase.Notifications.Discord.DaaSignal,
 
 config :sanbase, Sanbase.Notifications.Discord.ExchangeInflow,
   webhook_url: {:system, "EXCHANGE_INFLOW_DISCORD_WEBHOOK_URL"},
-  trading_volume_threshold: {:system, "EXCHANGE_INFLOW_TRADING_VOLUME_THRESHOLD", "1000000"},
+  trading_volume_threshold: {:system, "EXCHANGE_INFLOW_TRADING_VOLUME_THRESHOLD", "100000"},
```
Comment (tspenov) [inline config/notifications_config.exs]: can we add a `100000` threshold also for DAA  signal in this PR ?

Comment (tspenov) [reply config/notifications_config.exs]: added in another PR

**Comment by tspenov** [comment]
looks good 👍

**Pull request merged**